docs/generator.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+	"github.com/resumic/schema/docs/util"
+	"github.com/tidwall/gjson"
+)
+
+func main() {
+	var conf, doc, tmpl string
+	flag.StringVar(&conf, "conf", "schema.json", "Schema file")
+	flag.StringVar(&doc, "doc", "docs/dev/layer.md", "Markdown file")
+	flag.StringVar(&tmpl, "tmpl", "docs/dev/doc.md", "Output file")
```

Comment:
Seems like dev/doc.md is the actual doc and layer is the template.
Probably necessary to make that clear and instead of layer you could use something like dev/doc.tmpl

Reply:
Thanks noted!

Reply:
Also the naming of the variables seems switched.

Reply:
@stp-ip right!.Done on 4cc4551